packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -419,3 +419,36 @@ export class RelativeDateFormatter {
     return I18nUtils.format(label, relativeDate.amount);
   }
 }
+
+export class Store {
+
+  static initialize() {
+    return { 
+      state : { 
+        facets: {},
+        numericFacets: {},
+        dateFacets: {},
+        timeFrameFacets: {},
```

Comment:
So far, the Timeframe facet seems to be handled as a Date facet by headless. We might want to register Date facets, Timeframe facets, and Date filters into the same `dateFacets` store property.